File: 2714-left-and-right-sum-differences/left-and-right-sum-differences.java
class Solution {
    public int[] leftRightDifference(int[] nums) {
        int lsum = 0, rsum = 0;
        for(int i = 0; i < nums.length; i++){
            rsum += nums[i];
        }
        int output = 0;
        for(int i = 0; i < nums.length; i++){
            rsum -= nums[i];
            output = Math.abs(lsum - rsum);
            lsum += nums[i];
            nums[i] = output;

        }
        return nums;
    }
}

File: 2714-left-and-right-sum-differences/left-and-right-sum-differences.js
var leftRightDifference = function(nums) {
    let lsum = 0, rsum = 0;
    let output = [];

    for(let i = 0; i < nums.length;i++){
        rsum += nums[i];
    }
    
    for(let i = 0; i < nums.length; i++){
        rsum -= nums[i];
        output.push(Math.abs(lsum - rsum));
        lsum += nums[i];
    }
    return output;
    
};